js/okex.js

```diff
@@ -1593,9 +1593,24 @@ module.exports = class okex extends Exchange {
             const code = this.safeCurrencyCode (id);
             const balance = this.safeValue (info, id, {});
             const account = this.account ();
+            const totalAvailBalance = this.safeFloat (balance, 'total_avail_balance');
+            if (this.safeString(balance, 'margin_mode') === 'fixed') {
+                const contracts = this.safeValue(balance, 'contracts', []);
+                let free = totalAvailBalance;
+                for (let i = 0; i < contracts.length; i++) {
+                    let cont = contracts[i];
+                    free += this.safeFloat (cont, 'fixed_balance') + this.safeFloat (cont, 'realized_pnl') 
```

Comment:
We have to implement the `+` operations with `this.sum`, as explained here:

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#adding-numbers-and-concatenating-strings

Reply:
Im sorry, I can't pull the reposition to my pc, so can't run eslint to check.

Reply:
I'd suggest to split line 1602 into basic statements:

```JavaScript
free += this.safeFloat (cont, 'fixed_balance') + this.safeFloat (cont, 'realized_pnl') - this.safeFloat (cont, 'margin_frozen') - this.safeFloat (cont, 'margin_for_unfilled');
```

↓

```JavaScript
const fixedBalance = this.safeFloat (cont, 'fixed_balance');
const realizedPnl = this.safeFloat (cont, 'realized_pnl');
const marginFrozen = this.safeFloat (cont, 'margin_frozen');
const marginForUnfilled = this.safeFloat (cont, 'margin_for_unfilled');
const margin = this.sum (fixedBalance, realizedPnl) - marginFrozen - marginForUnfilled;
free = this.sum (free, margin);
```